string-manipulation-2/src/swap-chars.js

```diff
@@ -0,0 +1,11 @@
+/* exported swapChars */
+
+function swapChars(firstIndex, secondIndex, string) {
+  var firstChar = string.slice(firstIndex, firstIndex + 1);
+  var secondChar = string.slice(secondIndex, secondIndex + 1);
```

Comment:
You can access characters in a string by their index. `var firstChar = string[firstIndex];`

See if that helps you cut down on your slicing a bit.